pkg/deploy/generator/resources_rp.go

```diff
@@ -1109,6 +1109,34 @@ func (g *generator) database(databaseName string, addDependsOn bool) []*arm.Reso
 		Type: "Microsoft.DocumentDB/databaseAccounts/sqlDatabases",
 	}
 
+	mimo := &arm.Resource{
```

Comment:
Q: so the `pkg/deploy/assets/databases-development.json` file is generated by this generator function?